### PR TITLE
Cleanup & Refactor

### PR DESCRIPTION
**Pull request by ivan-c**
* Simplify XPath expressions
* Refactor FHIR modeling
* Remove erroneous/extra `medicationCodeableConcept` wrapping from MedicationOrder

### CODE REVIEW

**Comment by ivan-c** [comment]
>Remove erroneous/extra medicationCodeableConcept wrapping from MedicationOrder

@achen2401 FYI this will require a change in the AHRQ/frontend

**Comment by achen2401** [comment]
@ivan-c Ivan thank you very much for the heads-up. Note to self, fix here:
https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY/blob/af8fe1c9b9e5dbe3f72215a4e0714b46e40c9980/src/components/summary.json#L103

**Review by achen2401 — APPROVED** [review]

File: script_facade/client/client_106.py [hunk 1]
```diff
@@ -94,11 +96,11 @@ def parse_patient_lookup_query(xml_string):
     root = ET.fromstring(xml_string.encode('utf-8'))
 
     # todo: use SCRIPT XML namespace correctly
-    patient_elements = root.xpath('//*[local-name()="Patient"]')
+    patient_elements = root.xpath('//script:Patient', namespaces=SCRIPT_NAMESPACE)
```
Comment (pbugni) [inline script_facade/client/client_106.py:97]: doesn't this mean you can remove the todo on the previous line?

File: script_facade/models/r1/medication_order.py [hunk 2]
```diff
@@ -8,56 +8,75 @@
...
+        self.medication = None
 
-    def __init__(self, med, dispense_request=None, prescriber=None, date_written=None, date_ended=None):
-        # todo: support medicationReference and medicationCodeableConcept
-        self.medication = med
-        self.dispense_request = dispense_request
-        self.prescriber = prescriber
-
-        self.date_written = date_written
+        self.date_written = None
```
Comment (pbugni) [inline script_facade/models/r1/medication_order.py:16]: I'm sure you mean to take the values given.  i.e. `self.date_written = date_written` (which will be None if the caller doesn't include a better `date_written` param).

**Review by pbugni — CHANGES_REQUESTED** [review]

File: script_facade/models/r1/medication_order.py (hunk 2 above)
Comment (ivan-c) [reply script_facade/models/r1/medication_order.py:16]: I'll remove this line, and the lines the follow, that set [MedicationOrder](https://cosri-pdmp.cirg.washington.edu/v/r2/fhir/MedicationOrder) attributes (`date_written`, `date_ended`, `dispense_request`, `prescriber`) that aren't required by FHIR.

...or is it better to initialize these all to `None`, so I don't have to check if each instance has an attribute set before accessing it?...

**Review by pbugni — APPROVED** [review]

File: script_facade/models/r1/medication_order.py (hunk 2 above)
Comment (ivan-c) [reply script_facade/models/r1/medication_order.py:16]: after discussion, setting all attributes to `None`

**Pull request merged**